### ISSUE TITLE
Module: bigip_sslo_config_topology - pool selection for an inbound L3 topology doesn't seem to work

### ISSUE THREAD

**Issue by kevingstewart** [F5Networks/f5-ansible-bigip#27]
##### COMPONENT NAME
Ansible module: bigip_sslo_config_topology

##### ANSIBLE VERSION
```
ansible [core 2.12.4]
  config file = /sslo/ansible/ansible.cfg
  configured module search path = ['/sslo/ansible/library']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  ansible collection location = /sslo/ansible/collection
  executable location = /usr/local/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.1
  libyaml = True
```

##### BIGIP VERSION
```
Sys::Version
Main Package
  Product     BIG-IP
  Version     15.1.1
  Build       0.0.6
  Edition     Point Release 0
  Date        Thu Oct  8 02:52:59 PDT 2020
```

##### CONFIGURATION
ansible.cfg:
```
[defaults]
host_key_checking = False
retry_files_enabled = False
inventory           = ./inventory/hosts
library             = ./library
roles_path          = ./roles
collections_paths   = ./collection
```

##### OS / ENVIRONMENT
Running Ansible inside Ubuntu:20.04 Docker container

##### SUMMARY
Pool assignment for an inbound L3 topology doesn't seem to work. No pool is assigned to the topology VIP.

##### STEPS TO REPRODUCE
Using this declaration:
```
- name: Create SSLO Topology
  bigip_sslo_config_topology:
    name: "l3inboundapp"
    topology_type: "inbound_l3"
    dest: "10.0.2.200/32"
    port: 443
    ssl_settings: "sslconfig"
    security_policy: "ssloP_sslopolicy"
    vlans:
    - "/Common/external"
    snat: "automap"
    pool: "/Common/webapp"
```

##### EXPECTED RESULTS
webapp pool should be assigned to the topology. Also, as a function of an "application mode" inbound topology, a pool is assigned and address/port translation are enabled.

##### ACTUAL RESULTS
webapp pool is not assigned to the topology, and address/port translation are not enabled.

**Comment by trinaths** [comment]
Created [INFRAANO-742] for internal tracking.

**Comment by wojtek0806** [comment]
fixed in 1.8 release
